backend/src/models/pairing.ts

```diff
@@ -0,0 +1,36 @@
+import mongoose from "mongoose";
+
+interface PairingInterface {
+    menteeId: string;
+    mentorId: string;
+    whyPaired: string;
```

Comment:
Regarding whyPaired - this is more of a design question as well, but does whyPaired not need to be different for the mentor and mentee?For instance the mentor might see "this mentee likes your interests" and the mentee might see "this mentor is in your desired career field." In this case I believe we might need 2 separate whyPaired params. If not then this looks good!

Reply:
That's a great point! I think they should be common but lemme check with @mrinals108